test/parallel/test-http-mutable-headers.js

```diff
@@ -150,7 +150,7 @@ const s = http.createServer(common.mustCall((req, res) => {
     case 'writeHead':
       res.statusCode = 404;
       res.setHeader('x-foo', 'keyboard cat');
-      res.writeHead(200, { 'x-foo': 'bar', 'x-bar': 'baz' });
+      res.writeHead(200, undefined, { 'x-foo': 'bar', 'x-bar': 'baz' });
```

Comment:
Should we also keep the test case for `writeHead(status[, headers])`?

Reply:
Yes, I'm looking for a suitable test file to add this new case

Reply:
@Himself65 ping

Reply:
oh no, I have forgot about this...